R/data_rename.R

```diff
@@ -1,4 +1,5 @@
 #' @title Rename columns and variable names
+#' @name data_rename
```

Comment:
This is unnecessary since the function is already named `data_rename`

Reply:
I know, but I prefer having a consistent structure of the header part of an R file, including explicit tags (else, it depends on the order, which _may_ change).